R/zchunk_L202.an_input.R

```diff
@@ -0,0 +1,646 @@
+    L202.StubTech_in %>%
+      filter(!region %in% aglu.NO_AGLU_REGIONS) %>%
+      anti_join(L202.no_ddgs_regions_subs, by = c("region", "supplysector", "subsector", "stub.technology" = "technology")) ->
+      L202.StubTech_in
+    L202.StubTechInterp_in %>%
+      filter(!region %in% aglu.NO_AGLU_REGIONS) %>%
+      anti_join(L202.no_ddgs_regions_subs, by = c("region", "supplysector", "subsector", "stub.technology" = "technology")) ->
+      L202.StubTechInterp_in
+    L202.StubTechProd_in %>%
+      filter(!region %in% aglu.NO_AGLU_REGIONS) %>%
+      anti_join(L202.no_ddgs_regions_subs, by = c("region", "supplysector", "subsector", "stub.technology" = "technology")) ->
+      L202.StubTechProd_in
+
+    L202.Supplysector_an <- filter(L202.Supplysector_an, !region %in% aglu.NO_AGLU_REGIONS)
+    L202.SubsectorAll_an <- filter(L202.SubsectorAll_an, !region %in% aglu.NO_AGLU_REGIONS)
+    L202.StubTech_an <- filter(L202.StubTech_an, !region %in% aglu.NO_AGLU_REGIONS)
+    L202.StubTechInterp_an <- filter(L202.StubTechInterp_an, !region %in% aglu.NO_AGLU_REGIONS)
+    L202.StubTechProd_an <- filter(L202.StubTechProd_an, !region %in% aglu.NO_AGLU_REGIONS)
+    L202.StubTechCoef_an <- filter(L202.StubTechCoef_an, !region %in% aglu.NO_AGLU_REGIONS)
+    L202.StubTechFixOut_imp_an <- filter(L202.StubTechFixOut_imp_an, !region %in% aglu.NO_AGLU_REGIONS)
+
+    # Produce outputs
+    L202.RenewRsrc %>%
+      add_title("Generic resource attributes") %>%
+      add_units("NA") %>%
+      add_comments("A_agRsrc written to all regions") %>%
+      add_legacy_name("L202.RenewRsrc") %>%
+      add_precursors("aglu/A_agRsrc", "common/GCAM_region_names") ->
+      L202.RenewRsrc
+
+    L202.RenewRsrcPrice %>%
+      add_title("Resource prices") %>%
+      add_units("1975$/kg") %>%
+      add_comments("A_agRsrc written to all regions with `year` set to first model base year and `price` = 1") %>%
+      add_legacy_name("L202.RenewRsrcPrice") %>%
+      add_precursors("aglu/A_agRsrc", "common/GCAM_region_names") ->
+      L202.RenewRsrcPrice
+
+    L202.maxSubResource %>%
+      add_title("Maximum amount of resource production allowed in any period") %>%
+      add_units("Mt/yr") %>%
+      add_comments("Computed as the maximum of all base periods, for each region and resource") %>%
+      add_legacy_name("L202.maxSubResource") %>%
+      add_precursors("L109.an_ALL_Mt_R_C_Y", "L108.ag_Feed_Mt_R_C_Y", "aglu/A_agRsrcCurves", "common/GCAM_region_names") ->
+      L202.maxSubResource
+
+    L202.RenewRsrcCurves %>%
+      add_title("Renewable resource curves") %>%
+      add_units("units") %>%  # TODO
+      add_comments("A_agRsrcCurves written to all regions") %>%
+      add_legacy_name("L202.RenewRsrcCurves") %>%
+      add_precursors("aglu/A_agRsrcCurves", "common/GCAM_region_names") ->
+      L202.RenewRsrcCurves
+
+    L202.UnlimitedRenewRsrcCurves %>%
+      add_title("Unlimited renewable resource curves") %>%
+      add_units("units") %>%   # TODO
```

Comment:
This is unitless (the only data values are for the capacity factor, which doesn't apply to these commodities)